src/compiler/crystal/annotatable.cr

```diff
@@ -19,5 +19,10 @@ module Crystal
     def annotations(annotation_type : AnnotationType) : Array(Annotation)?
       @annotations.try &.[annotation_type]?
     end
+
+    # Returns all annotations on this type, if any, or `nil` otherwise
+    def all_annotations : Array(Annotation)?
```

Comment:
Is there a reason this isn't called just `annotations`?

Reply:
That's a good call. Will fit better with the existing annotations methods. I'll have to play around with it as it'll conflict with the `@annotations` property. But should be easy enough to rename that to something else as its internal only.

Reply:
Oh, I didn't notice this was the Crystal side. I was thinking about the API name.

Reply:
Still worth doing. It's a small enough change and keeps everything consistent.

Reply:
Hm, I'm not so sure about this. I think calling the new method `all_annotations` would be better than renaming the property to `applied_annotations` which looks a bit odd.

Reply:
There's just no real gain from this big renaming everywhere.

Reply:
Actually yea that's a good point. We can still keep the public API method `#annotations`, but just make it `#all_annotations` internally. Does break the consistency of things but not a huge deal and probably a bit more clear anyway. I can deal with that tonight.